lefthook.yml

```
@@ -11,3 +11,27 @@ pre-push:
+          echo ""
+          echo "✅ Examples:"
+          echo "  feat: ~~기능을 개발했어요"
+          echo "  fix: ~~버그를 고쳤어요"
+          echo "  docs: 문서를 수정했어요"
+          echo "  style: 코드 포맷을 변경했어요"
+          echo "  refactor: 기능 변경을 하지 않고 코드를 개선했어요"
+          echo ""
+          echo "👀 사용 가능한 옵션이에요:"
+          echo "  feat, fix, docs, style, refactor, perf, test, build, ci, chore, revert"
+          exit 1
+        }
```

yamllint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)